conductor_api/src/interface.rs

```diff
@@ -138,8 +138,18 @@ impl ConductorApiBuilder {
             let hc_lock = instance.clone();
             let hc_lock_inner = hc_lock.clone();
             let mut hc = hc_lock_inner.write().unwrap();
-            let call_params = params_map.get("params");
-            let params_string = serde_json::to_string(&call_params)
+
+            // Getting the arguments of the call contained in the json-rpc 'params'
+            let call_args = match params_map.get("args") {
+                Some(args) => Some(args),
+                None => {
+                    // TODO: Remove this fall back to the previous impl of inner 'params' 
+                    // as soon as its deprecation life cycle is over <17-04-19, yourname> //
+                    hc.context().log("warn/interface: DEPRECATION WARNING: Using 'params' instead of 'args' for a Zome function call is now deprecated.");
```

Comment:
@dymayday is there other places in the code where we use this `warn` prefix in the logging?

Reply:
Actually @maackle suggested it to me earlier :)
https://github.com/holochain/holochain-rust/pull/1271#discussion_r276430179

Reply:
Ok cool. I haven't seen it around, but I trust! Just wondering :)